src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -51,10 +50,7 @@ const Conversation: React.FC<ConversationProps> = props => {
 
   const isArtworkOfferable = !!artwork && !!artwork?.isOfferableFromInquiry
 
-  const isOfferable =
-    user &&
-    userHasLabFeature(user, "Web Inquiry Checkout") &&
-    isArtworkOfferable
+  const isOfferable = user && isArtworkOfferable
```

Comment:
We only needed the user here to satisfy a type error that was being thrown we used `userHasLabFeature`. Since we are removing that I think we can refactor this to the following:

```ts
const isOfferable = !!artwork && !!artwork?.isOfferableFromInquiry
```

Reply:
It also might be a good idea (and fun) to clean up the lab feature in gravity. Let me know if you want to pair on that.

Reply:
Gotcha. Comment addressed.
Yeah, just to make sure we get the feature out and that this gets deployed first I wanted to push this through, and then deal with the gravity cleanup 😊